tests/models/reservoirarea.csv

```diff
@@ -0,0 +1,3 @@
+volumes,values
```

Comment:
I think this is formatted incorrectly. Shouldn't it be?

```csv
volumes,values
0,0
200,100
400,200
600,300
800,400
1000,500
```

Reply:
totally right!